node/libs/concurrency/Cargo.toml

```diff
@@ -18,3 +18,43 @@ tokio.workspace = true
 tracing.workspace = true
 tracing-subscriber.workspace = true
 vise.workspace = true
+
+[lints.rust]
```

Comment:
Do I understand correctly that the only reason not to use `lints.workspace = true` here is to override `unsafe_code` setting? I'd suggest to demote this lint to the `deny` level; then, it could be overridden on the module level.

Reply:
You are indeed right. I'm not sure your suggestion works, let me try it.

Reply:
I've tried it locally before suggesting 🙃